.github/workflows/check-release.yml

```diff
@@ -21,16 +16,19 @@ jobs:
       - name: Install Conda environment with Micromamba
         uses: mamba-org/setup-micromamba@v1
         with:
-          environment-name: voici
+          environment-name: voici-dev
           create-args: >-
             python=3.9
             yarn=3
+            pip
+            jupyter-releaser
 
       - name: Check Release
-        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@main
```

Comment:
Looks like https://github.com/jupyter-server/jupyter_releaser/pull/549 was released as part of https://github.com/jupyter-server/jupyter_releaser/releases/tag/v1.6.0, so we can stick to `v2`.

Reply:
Right! Thank you I'll make the change in the coming PR